.gitignore

```diff
@@ -29,7 +29,7 @@ disabledPlugins
 webpackstats.json
 /config/*
 !/config/kibana.yml
-!/config/apm.js
+!/config/apm.dev.js
```

Comment:
I don't think it's a good idea to allow someone to accidentally check their `apm.dev.js` file into git. That's why I left it out. What is the motivation behind this change?

Reply:
You are right, I went with this https://github.com/elastic/kibana/pull/44281#discussion_r357701994, but forgot that we shouldnt allow devs to check in, I am reverting this change now.